editoast/src/views/infra/mod.rs

```diff
@@ -255,7 +255,18 @@ pub enum InfraState {
     Error,
 }
 
-#[derive(Debug, Clone, Serialize, ToSchema)]
+impl From<editoast_osrdyne_client::WorkerStatus> for InfraState {
+    fn from(status: editoast_osrdyne_client::WorkerStatus) -> Self {
+        match status {
+            editoast_osrdyne_client::WorkerStatus::Unscheduled => InfraState::NotLoaded,
+            editoast_osrdyne_client::WorkerStatus::Started => InfraState::Initializing,
+            editoast_osrdyne_client::WorkerStatus::Ready => InfraState::Cached,
+            editoast_osrdyne_client::WorkerStatus::Error => InfraState::Error,
+        }
```

Comment:
You can remove the outdated `InfraState` variants

Reply:
If that's okay with you, I'd rather do this all at once when we actually integrate the new loading states (from k8s)